pkg/workflow/controller/client.go

```diff
@@ -0,0 +1,7 @@
+package controller
+
+import "github.com/caicloud/cyclone/pkg/k8s/clientset"
+
+var (
+	Client clientset.Interface
```

Comment:
Please try to avoid define this global variable here.

Reply:
Is there any other way to create a util client ?